src/utils/issue.ts

```diff
@@ -174,7 +171,9 @@ export async function getAvailableOpenedPullRequests(context: Context, username:
 
   for (let i = 0; i < openedPullRequests.length; i++) {
     const openedPullRequest = openedPullRequests[i];
-    const reviews = await getAllPullRequestReviews(context, openedPullRequest.number);
+    const owner = openedPullRequest.html_url.split("/")[3];
+    const repo = openedPullRequest.html_url.split("/")[4];
```

Comment:
Can be done in one instruction. What happens if the url is not from an organization?